malariagen_data/veff.py

```diff
@@ -27,19 +25,14 @@ def __init__(self, genome, geneset):
         self._genome_cache = dict()
         self._geneset_cache = None
 
-        geneset.rename(
-            columns={"ID": "feature_id", "Parent": "parent_id", "end": "stop"},
-            inplace=True,
-        )
```

Comment:
I thought it would be simpler to do away with the renaming altogether, just use columns as named.